content/applications/inventory_and_mrp/purchase/advanced/vendor_costs_report.rst

```diff
@@ -0,0 +1,119 @@
+
+Add measures
+------------
+
+After selecting the :guilabel:`Filters`, :guilabel:`Group by`, and :guilabel:`Comparison` settings,
+click out of the drop-down menu.
+
+By default, the report displays with the following measures: :guilabel:`Order`, :guilabel:`Total`,
+:guilabel:`Untaxed Total`, and :guilabel:`Count`. Click :guilabel:`Measures` at the top-left to open
+the drop-down list of available measures. Click :guilabel:`Average Cost` to add it to the report.
+Select any additional measures to add to the report, or click on any of the already selected
+measures to remove them, if desired.
+
+.. tip::
+   It is recommended to run the report with at least :guilabel:`Average Cost`, :guilabel:`Total`, or
+   :guilabel:`Untaxed Total` selected from the :guilabel:`Measures` list. Additional measures, such
+   as :guilabel:`Days to Receive`, can be added to provide additional insights.
+
+View results
+============
+
+The vendor costs report is displayed in the *pivot* view, by default. Click :guilabel:`Insert in
+Spreadsheet` to add the pivot view into an editable spreadsheet format within the *Dashboards* app.
+If the Odoo *Documents* app is installed, the report can be inserted into a blank or existing
+spreadsheet, and exported.
+
+.. image:: vendor_costs_report/sample-vendor-report.png
+   :align: center
+   :alt: A sample of a vendor costs report with the measures set as total and average costs.
+
+.. note::
+   The vendor costs report is also available in *graph* view. Click the :icon:`fa-area-chart`
+   :guilabel:`(area chart)` icon to change to graph view. Click the corresponding icon at the top of
+   the report to switch to a :icon:`fa-bar-chart` :guilabel:`(bar chart)`, :icon:`fa-line-chart`
+   :guilabel:`(line chart)`, or :icon:`fa-pie-chart` :guilabel:`(pie chart)`.
+
+Save and share a favorite search
+--------------------------------
+
+The *Favorites* feature found on Odoo reports allows users to save their most commonly used filters,
+without having to reconstruct them every time they are needed.
+
+To create and save a new *Favorites* configuration on a report, follow the steps below:
+
+#. Set the necessary parameters using the :guilabel:`Filters`, :guilabel:`Group By` and
+   :guilabel:`Measures` options.
+#. Click the :icon:`fa-caret-down` :guilabel:`(down)` icon next to the :guilabel:`Search...` bar to
+   open the drop-down menu.
+#. Under the :guilabel:`Favorites` heading, click :guilabel:`Save current search`.
+#. If desired, enter a new name for the report.
+#. Tick the :guilabel:`Default Filter` checkbox to have these filter settings automatically
+   displayed when the report is opened. Otherwise, leave it blank.
+#. Tick the :guilabel:`Shared` checkbox to make this filter configuration available to all other
+   database users. If this checkbox is not ticked, only the user who created the filter can access
+   it.
+#. Click :guilabel:`Save` to preserve the configuration for future use.
```

Comment:
Suggestion to just link to the Odoo essentials documentation for creating a Favorite search:
https://www.odoo.com/documentation/17.0/applications/essentials/search.html#favorites
```
:ref:`search/favorites`
```